easybuild/easyconfigs/b/BamTools/BamTools-2.5.1-foss-2018a.eb

```diff
@@ -0,0 +1,19 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+name = 'BamTools'
+version = '2.5.1'
+
+homepage = 'https://github.com/pezmaster31/%(namelower)s'
+description = "BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files."
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/pezmaster31/%(namelower)s/archive']
+sources = ['v%(version)s.tar.gz']
+
+checksums = ['4abd76cbe1ca89d51abc26bf43a92359e5677f34a8258b901a01f38c897873fc']
+
+builddependencies = [('CMake', '3.9.5')]
```

Comment:
@Serpens Please bump `CMake` version to `3.10.2` (we use this version in all `2018a` easyconfigs currently in `develop`